server.py

```diff
@@ -96,9 +114,8 @@ def handle_playlist(playlist_url:str):
     playlist = Playlist(playlist_url)
     # Update process state
     if State.INTERLUDE in process_dict:
-        interlude_process = process_dict.pop(State.INTERLUDE)
+        stop_video_by_type[State.INTERLUDE]
```

Comment:
i believe this should be a function call, we are using `[]` here

Reply:
done